common_examples/tcp_client/README.md

```diff
@@ -0,0 +1,20 @@
+# TCP client example
+
+This example provides a basic implementation of tcp client for ESP32 with drivers from this repository
+
+## About the example
+TCP client is designed to periodically transmit to the server, and accept responses, which are printed to the console.
+
+## Configuring the example
+Configure the example using `idf.py menuconfig`, according to your setup, provide settings for Ethernet initialization. Next go to `Example options` and input the IP address of your PC in the local networkto wich ESP32 is connected. 
+
+## Running the example
+
+Ater you obtain an IP address you will see message reading `TCP client is started, waiting for the server to accept a connection.`. 
+
+Start the server with `tcp_server.py`. Command to do so is `tcp_server.py IP` to start listening on specified IP address.
```

Comment:
This is just a one of the options, am I right? Also mention the option with running the server in other ESP. You can briefly mention it and provide link to the Server example.

Reply:
Added the code required and modified the readme.